### PR TITLE
Join game modal improvements

### PR DESCRIPTION
**Pull request by Vasso**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,7 @@ build/Release
 # Deployed apps should consider commenting this line out:
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules
+bower_components
```
Comment (artyomtrityak) [inline .gitignore]: revert it please, components should be added

File: client/static/app/modules/play/controllers/play-new.controller.js [hunk 2]
```diff
@@ -34,15 +34,16 @@ define(function(require) {
...
-        })
-        .catch(function() {
-          vm.state.isLoading = false;
-        });
+      .then(function(play) {
+        vm.state.isLoading = false;
+        $rootScope.$emit('dg:plays:reload');
+        $modalInstance.close();
+        $location.search('playId', play._id);
+        dgPlayService.join(play._id, play.creator);
```
Comment (artyomtrityak) [inline client/static/app/modules/play/controllers/play-new.controller.js]: why are you calling this here? If this is a query you should close modal / do search etc only after this query will be done, so `return dgPlayService.join(play._id, play.creator);` and do all actions in next `then`

File: client/static/app/modules/play/templates/play-join.tpl.html [hunk 3]
```diff
@@ -1,22 +1,48 @@
...
-                <div class="text-center">
-                    <img class="img-thumbnail img-responsive" ng-src="{{joinIns.playData.img}}" src="http://placehold.it/290" width="290" height="290">
+            <div class="col-md-8">
+                <div class="row m-b-sm">
+                    <div class="col-md-3">
+                        <img class="img-thumbnail img-responsive" ng-src="{{joinIns.playData.img}}" src="http://placehold.it/100">
+                    </div>
+                    <div class="col-md-9">
+                        <h3>{{joinIns.playData.game.nameOrigin}}</h3>
+                        <p><b>Где</b>: <a target="_blank" href="https://www.google.com.ua/maps/place/{{joinIns.playData.city}}, {{joinIns.playData.address}}">{{joinIns.playData.city}}, {{joinIns.playData.address}}</a></p>
```
Comment (artyomtrityak) [inline client/static/app/modules/play/templates/play-join.tpl.html]: it should be ng-href

File: server/modules/plays/model.js [hunk 4]
```diff
@@ -9,15 +9,15 @@ var moment = require('moment'),
 // TODO: set correct validation
 // Real types like Number instead of Mixed makes undefined values invalid
 var PlaySchema = new Schema({
-  name: {
+  /*name: {
```
Comment (artyomtrityak) [inline server/modules/plays/model.js]: why?

Comment (Vasso) [inline server/modules/plays/model.js]: we don't have name field, only game

**Pull request merged**